WoWPro_Leveling/Wrath/Alliance/VAN_WOTLK_19_25_Alliance.lua

```diff
@@ -125,15 +125,15 @@ T Mathias and the Defias|QID|2360|M|68.4,70.2|Z|Westfall|N|To Agent Kearnen. You
 A Klaven's Tower|QID|2359|M|68.4,70.2|Z|Westfall|N|From Agent Kearnen. You will need at least 70 in Lockpicking to complete this quest.|C|Rogue|PRE|2360|
 C Klaven's Tower|QID|2359|M|70,75|Z|Westfall|QO|2|N|Pickpocket from the Malformed Defias Drone that roams around. If he doesn't have it you have to wait for him to despawn and respawn.|C|Rogue|
 C Klaven's Tower|QID|2359|M|70.4,73.93|Z|Westfall|QO|1|N|Sneak to the top of the tower, sap the boss and lockpick the box.|C|Rogue|
-F Stormwind City|QID|2359|M|56.55,52.65|Z|Westfall|N|Fly or run to Stormwind.|C|Rogue|P|Lockpicking;633;0+70;0|
+f Stormwind|QID|2359|M|56.55,52.65|Z|Westfall|N|Fly or run to Stormwind.|C|Rogue|P|Lockpicking;633;0+70;0|
```

Comment:
lower case f is to pick up a flightpath, not to use the Flightpath to fly. It needs to be F

Reply:
You are correct, been a while sinc ei made the flightpoint changes, so don't remember why. I'll check them all again to see if there are more weird things. Might've been caused by a Find-replace and me not paying good enough attention.

Reply:
no problem